src/Tribe/Context/locations.php

```diff
@@ -76,6 +76,25 @@
 	],
 	'post_type' => [
 		'read' => [
+			Tribe__Context::FUNC        => static function() {
+				$post_type_objs = get_post_types( [
+					'public' => true,
+					'_builtin' => false,
+				], 'objects' );
+
+				foreach( $post_type_objs as $post_type ) {
+					if ( empty( $post_type->query_var ) ) {
+						continue;
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: ff617c134127af27459182ad52911af6</sub>